### PR TITLE
Improve translations and usability

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this patch includes a whole bunch of improvements to the translations, accompanied by some humble usability improvements.

With kind regards,
Andreas.

### CODE REVIEW

File: app/scripts/controllers/feed.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ const feed = angular.module('feed', ['stats', 'udc', 'utils'])
     // Set default menu data
     $scope.menu = [{
       'url': 'https://crate.io/demo?utm_source=adminui&utm_medium=browser&utm_term=&utm_content=demolink&utm_campaign=newsfeed&ajs_event=clicked_demo_link',
-      'title': 'Schedule a 1-ON-1 demo with a Crate.io engineer'
+      'title': 'Schedule a 1-ON-1 demo with a Crate engineer'
```
Comment (seut) [inline app/scripts/controllers/feed.js:130]: why that change? at the translation files you wrote `Crate.io ....` (https://github.com/crate/crate-admin/pull/712/files#diff-cfbca3bbdba4f816f75a9bb32f41f9dbc41ce1b20c342b65bab0337e491dbec2R193) Should be consistent at least ;)

File: app/styles/views/_console.scss [hunk 2]
```diff
@@ -96,7 +97,7 @@
 .cr-console-header__options__item--hint{
 	@extend .cr-console-header__options__item;
 	border-left: none;
-	justify-content: flex-end;
+	/* justify-content: flex-end; */
```
Comment (seut) [inline app/styles/views/_console.scss:100]: no needed? so maybe delete?

**Review by seut — APPROVED** [review]
Lgtm otherwise

File: app/styles/views/_console.scss (hunk 2 above)
Comment (amotl) [reply app/styles/views/_console.scss]: Deleted.

**Pull request merged**